handlers/movieHandler.go

```diff
@@ -0,0 +1,29 @@
+package handlers
```

Comment:
Since these files are under handlers/ anyway, we needn't add the handler suffix in the file names. Use handlers/movies.go, handlers/ping.go

Reply:
I pushed the [changes](https://github.com/ashwinspg/explore-golang/pull/2/commits/3d4ca6ba4e83140b3309d987857398ea386775fa)